package.json

```diff
@@ -69,15 +69,18 @@
   "scripts": {
     "clean": "bsb -clean",
     "build": "bsb -make-world",
-    "watch": "tsc -b -w",
+    "watch:bsb": "bsb -make-world -w",
+    "watch:tsc": "tsc -b -w",
```

Comment:
We dont use tsc anywhere? Did I miss anything?

Reply:
there is a bunch of TS files in the `client/src`:
https://github.com/arrowresearch/vscode-merlin/tree/master/client/src

Are those not used?

Reply:
No, we can remove them. Apologies, I forgot. Can you please remove them?

Reply:
You mean we can remove everything related to TS? I can do it in a separate PR if you want